helm/rpm-builder/templates/ansible-rpm-builder-runner-buildconfig.yaml

```diff
@@ -0,0 +1,32 @@
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  name: {{ .Values.buildConfig.name }}
+  namespace: {{ .Values.targetNamespace }}
+spec:
+  runPolicy: Serial
+  source:
+    type: Dockerfile
+    dockerfile: |
+      FROM {{ .Values.buildConfig.image.name}}:{{ .Values.buildConfig.image.tag }}
+      USER 0
+      RUN dnf install python3 platform-python-pip dbus-devel skopeo systemd-devel golang rpm-build git make -y && \
+          pip3 install --upgrade pip && \
+          pip3 install openshift && \
```

Comment:
Consider running all the `pip3 install` calls (aside from upgrading pip) in a single call. Do you see an issue with that?

Reply:
No issues. I'll merge the lines. Thanks!